authorization/role/service/role_management_service_blackbox_test.go

```diff
@@ -467,3 +549,9 @@ func (s *roleManagementServiceBlackboxTest) checkScopeBelongsToResourceType(db *
 	require.NotEmpty(s.T(), scopesReturned)
 	require.NoError(s.T(), err)
 }
+
+func validateIdentityRole(s *roleManagementServiceBlackboxTest, expected identityrole.IdentityRole, actual identityrole.IdentityRole) {
```

Comment:
This code is duplicated in a few places. Just move it to test package and reuse.